scripts/build_renewable_profiles.py

```diff
@@ -241,6 +241,11 @@
         codes = corine["distance_grid_codes"]
         buffer = corine["distance"]
         excluder.add_raster(snakemake.input.corine, codes=codes, buffer=buffer, crs=3035)
+    
+    if "ship_threshold" in config:
+        shipping_threshold=config["ship_threshold"]*8760*6 # approximation because 6 years of data which is hourly collected
+        func = functools.partial(np.less,shipping_threshold)
```

Comment:
```suggestion
        shipping_threshold = config["ship_threshold"] * 8760 * 6 # approximation because 6 years of data which is hourly collected
        func = functools.partial(np.less, shipping_threshold)
```